NashvilleHistory/www/js/controllers/AllPlacesCtrl.js

```diff
@@ -40,20 +51,36 @@ app.controller('AllPlacesCtrl', function($scope, $state, $q, AllPlacesFact){
 
   getAllPlaces();
 
+
+  function areMarkersBookmarked (){
+      BookmarkFact.getAllBookmarks(AuthFact.getUserId())
+      .then((bookmarks)=>{
+        console.log("bookmarked markers", bookmarks);
+        Object.keys(bookmarks).map((key)=>{
+          AllMarkers.forEach((marker, index)=>{
+            if (bookmarks[key].latitude === marker.latitude && bookmarks[key].longitude === marker.longitude){
```

Comment:
This check actually isn't as full-proof as I thought. There are a lot of art pieces that share the same lat/lons coordinates, so if I bookmark one of those art pieces, on refresh they all show up as having been bookmarked.